web/cobrands/northamptonshire/assets.js

```diff
@@ -209,19 +183,19 @@ var northants_road_defaults = $.extend(true, {}, fixmystreet.alloyv2_defaults, {
         field: 'asset_resource_id'
     },
     getUSRN: function(feature) {
-      return feature.fid;
+      return feature.attributes.asset_id;
```

Comment:
I think you can remove the getUSRN function, and add an `attribute: "asset_id"` in the `usrn:` key just above, now it's a standard way of doing it.